paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/FlinkConnectorOptions.java

```diff
@@ -87,6 +87,15 @@ public class FlinkConnectorOptions {
                                     + "for each statement individually by also considering the global configuration. "
                                     + "If user enable the scan.infer-parallelism, the planner will derive the parallelism by inferred parallelism.");
 
+    public static final ConfigOption<Integer> COMPACTION_PARALLELISM =
+            ConfigOptions.key("compaction.parallelism")
```

Comment:
Do you want to use it in many places? Or is it just for the unaware table? If it's just a standalone use, I suggest using the key starting with unaware